main.tf

```
@@ -1,9 +1,13 @@
 data "aws_caller_identity" "current" {}
 
+locals {
+  kms_key_id = (var.encryption_enabled && var.kms_master_key_id != "") ? var.kms_master_key_id : ""
```

```suggestion
  kms_key_id = var.encryption_enabled ? var.kms_master_key_id : ""
```
The above is the same as 
```
(var.encryption_enabled && var.kms_master_key_id != "") ? var.kms_master_key_id : ""
```

but shorter